src/background/followUser.js

```diff
@@ -1,3 +1,10 @@
+SW.methods.isUserStoreFull = function() {
+  if(SW.stores.userStore.length > 25) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Minor suggestion, you can simply write

    return SW.stores.userStore.lenth > 25;

Reply:
Will do.